packages/components/pages/f-checkout/src/components/Checkout.vue

```diff
@@ -332,6 +350,18 @@ export default {
                 [EventNames.FormValid]: this.submitCheckout,
                 [EventNames.FormInvalid]: this.onInvalidCheckoutForm
             };
+        },
+
+        placeOrderNotes () {
```

Comment:
This one could probably come from the store too, but I'm happy if you want to do that in a later PR

Reply:
It's probably going to be a quiet week next week looking at our Jira board, so I will save that for a further PR to have something to do. 😅